mlflow/pyfunc/__init__.py

```diff
@@ -1268,7 +1268,8 @@ def batch_predict_fn(pdf):
                 else:
                     row_batch_args = input_batch
 
-                yield _predict_row_batch(batch_predict_fn, row_batch_args)
+                if len(row_batch_args[0]) > 0:
```

Comment:
Is this the correct check to use if `row_batch_args` is an instance of `pd.Series` or `pd.DataFrame`? `len(dataframe[0])` seems to get the length of the first row, which may not exist if the dataframe is empty.

Reply:
Can we add tests for the empty partition case?

Reply:
Should be correct here. 
if row_batch_args is an instance of pd.Series or pd.DataFrame, then `row_batch_args` is set to be `(input_batch,)`, i.e., in any cases, `row_batch_args` is normalized into a tuple.

Reply:
test added.

Reply:
Thanks :)